### DISCUSSION TITLE
Luarocks can't find LUA_INCDIR even though it is very installed.

### DISCUSSION THREAD

**Discussion by MBPpp** [keplerproject/luarocks#1392] [Q&A]
Hello,
I am trying to install lcurses and after a lot of Googling I finally have everything properly installed.
BUT
it still doesn't recognize some of the files.
If I put
`luarocks install lcurses`
it spits back
`Error: Could not find header file for NCURSES
  No file ncurses.h in c:/external/include
  No file ncurses.h in c:/mingw/include
  No file ncurses.h in c:/windows/system32/include
You may have to install NCURSES in your system and/or pass NCURSES_DIR or NCURSES_INCDIR to the luarocks command.
Example: luarocks install lcurses NCURSES_DIR=/usr/local`
and if I then point it to the ncurses.h file (because it is installed, it's just in a subfolder below mingw/include) it spits back
`Error: Failed finding Lua header files. You may need to install them or configure LUA_INCDIR.`
even though the lua files are also where they should be.
Does anyone have any idea what the hell is going on?

**Comment by hishamhm** [comment]
Try setting NCURSES_INCDIR to the specific folder where ncurses.h is, like this:

If ncurses.h is in c:/some/folder/mingw/include/ncurses/ncurses.h, do:

```
luarocks install lcurses NCURSES_INCDIR=c:/some/folder/mingw/include/ncurses
```
